docs/tutorials/eda/eda-auto-target-analysis.md

```diff
@@ -0,0 +1,68 @@
+# EDA: Automated Target Variable Analysis
+
+In this section we explore automated dataset overview functionality.
+
+Automated target variable analysis aims to automatically analyze and summarize the variable we are trying to predict (
+label). The goal of this analysis is to take a deeper look into target variable structure and its relationship with
+other important variables in the dataset.
+
+To simplify outliers and useful patterns discovery. This functionality introducing components which allow generating
```

Comment:
"To simplify outliers and useful patterns discovery." is a sentence fragment. I don't know what you want to do with it, but as a reader it's not clear how to read this.

Also, "introducing" -> "introduces"

Reply:
fixed